api-docs/v2/api-operations/methods/GET_listRateLimits_v2_account_id.rst

```diff
@@ -0,0 +1,183 @@
+|                                |                      |time.                 |
++--------------------------------+----------------------+----------------------+
+|limits.rate.limit. \ **unit**   |String                |The type of unit for  |
+|                                |                      |the rate limit.  For  |
+|                                |                      |example, ``MINUTE``   |
+|                                |                      |or ``DAY``.           |
++--------------------------------+----------------------+----------------------+
+|limits.rate.limit. \            |Integer               |The time remaining    |
+|**remaining**                   |                      |for the limit in      |
+|                                |                      |units.                |
++--------------------------------+----------------------+----------------------+
+|limits.rate.limit. \ **value**  |Integer               |The maximum allowed   |
+|                                |                      |time in units.        |
++--------------------------------+----------------------+----------------------+
+|limits.rate.limit. \ **verb**   |String                |The HTTP operation.   |
++--------------------------------+----------------------+----------------------+
+|limits.rate. \ **regex**        |Regular Expression    |The regular expression|
+|                                |                      |that defines the      |
+|                                |                      |actual endpoints that |
+|                                |                      |are encompassed in the|
+|                                |                      |rate limit.           |
++--------------------------------+----------------------+----------------------+
+|limits.rate. \ **uri**          |URL                   |The endpoint being    |
+|                                |                      |limited.              |
++--------------------------------+----------------------+----------------------+
+
+**Example: List rate limits, response**
+
+.. code::  
+
+    HTTP/1.1 200 OK
+    Vary: Accept
+    Content-Type: application/json
+
+   {
+     "limits": {
+       "rate": [
+         {
+           "limit": [
+             {
+               "next-available": "2016-05-26T14:59:13.120Z",
+               "unit": "MINUTE",
+               "remaining": 420,
+               "value": 420,
+               "verb": "GET"
+             }
+           ],
+           "regex": ".*/v2/(\\d+/limits).*",
+           "uri": "/limits"
+         },
+         {
+           "limit": [
+             {
+               "next-available": "2016-05-26T14:59:13.119Z",
+               "unit": "MINUTE",
+               "remaining": 420,
+               "value": 420,
+               "verb": "GET"
+             }
+           ],
+           "regex": ".*/v2/(\\d+/pools).*",
+           "uri": "/pools"
```

Comment:
The example I have, doesn't have all four of these rate limits.  Can I get a new example?